wire-compiler/src/main/java/com/squareup/wire/CommandLineOptions.java

```diff
@@ -64,7 +66,8 @@
    * <pre>
    * java WireCompiler --proto_path=&lt;path&gt; --java_out=&lt;path&gt;
```

Comment:
All this Javadoc here is completely useless as this isn't a public type. We should merge this class with `WireCompiler`.